server/config/routes.js

```diff
@@ -951,7 +951,8 @@ exports.configure = function configure(app) {
   app.put('/lots/:uuid', lots.update);
   app.get('/lots/:uuid/assignments/:depot_uuid', lots.assignments);
   app.post('/lots/:uuid/merge', lots.merge);
-  app.get('/lot/dupes/:label?/:entry_date?/:expiration_date?/:initial_quantity?/:inventory_uuid?', lots.getDupes);
+  app.get('/lots_candidates/:inventory_uuid', lots.getCandidates);
```

Comment:
The root of this in an inventory item, right?  If so, I would prefer that we use the path `/inventory/:uuid/lot_candidates`.  It's more REST-y.

Reply:
I fixed the lot name issue and changed the URL per your suggestion.

Reply:
If you think it makes sense, I can move the lot candidate function to the inventory controller and service.  I have now left it in the lots controller and service.